File: c/exercise02/count_chars.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int main(void) {
  char sentence[50];
  printf("What is the input string? ");
  fgets(sentence, sizeof(sentence), stdin);

  if (strlen(sentence) == 0) {
    printf("Please enter something.\n");
    return EXIT_SUCCESS;
  }

  printf("\"%s\" has %lu characters.\n", sentence, strlen(sentence));

  return EXIT_SUCCESS;
}

File: javascript/exercise02/count-chars.js
import { input } from "pynput";

const value = await input("What is the input string? ");

if (value.trim().length === 0) {
	console.log("Please enter something.");
	process.exit();
}

const total = count_chars(value);
console.log(`"${value}" has ${total} characters.`);

/**
 * @param {string} sentence
 * @returns {number}
 */
function count_chars(sentence) {
	return sentence.length;
}
